### DISCUSSION TITLE
Robot Learning Support

### DISCUSSION THREAD

**Discussion by peterdavidfagan** [google/brax#302] [General]
Hi there,

I am looking to start running some simulated robot learning experiments. I am considering spending time exploring the use of brax as an alternative to other existing simulators. 

I wished to ask if there are limitations I should be aware of before investing time in using the brax library for robot learning experiments? I understand that this is a very high-level question, I intend to explore the library further but thought it would be worth asking up front in case there are major limitations I should consider (e.g. support for realistic graphics). 

Thanks in advance for any advice on this topic and I look forward to starting to explore this library more over the coming days.

**Comment by btaba** [comment]
Hi @peterdavidfagan ! Thanks for looking into brax! Yes, a major current limitation is that we don't generate pixels from state, although we'd like to have that implemented at some point. A key advantage is that brax is written in jax, so natively runs on hardware accelerators. The physics is similar to other engines (depending on the physics backend), especially in the next major release of brax which you can check out [here](https://github.com/google/brax/discussions/286).  Let us know if you have other questions!

**Reply by peterdavidfagan** [reply]
Thanks for getting back to me on this @btaba and addressing my question. This library is interesting to me, its not applicable for my current project (due to certain features that I require) but I think I will look to explore the library more and track development since it is potentially useful for future work. 

Thanks for being a responsive maintainer.
